<!-- -->
<launch>
    <node pkg="slam_fusion" type="filter_slam" name="filter_slam" output="screen" >
      <!-- topic name setup -->
      <param name="gps_input_topic" type="string" value="/pose_gps"/>
      <param name="gps_heading_topic" type="string" value="/heading_gps"/>
      <param name="imu_input_topic" type="string" value="/vehicle/imu"/>
      <param name="encoder_input_topic" type="string" value="/vehicle/speed_feedback"/>
      <param name="gps_output_topic" type="string" value="/slam_pose"/>
      <param name="yaw_output_topic" type="string" value="/slam_yaw"/>
      <param name="encoder_input_frequency" type="double" value="100"/>
      <param name="gps_input_frequency" type="double" value="10"/>
      <param name="imu_input_frequency" type="double" value="100"/>
      <!-- ***********the frequency of gps and yaw output************* -->
      <param name="output_frequency" type="double" value="100"/>
      <!-- *********************************************************** -->
      <!-- sensor error setup -->
      <!-- error of GPS(state:4) 0.008m 0.0005 -->
      <param name="GPS_error_fix" type="double" value="0.0025"/>
      <!-- error of GPS(state: 5) 1m -->
      <param name="GPS_error_float" type="double" value="1.0"/>
      <!-- error of GPS(state: 1) 3m -->
      <param name="GPS_error_single" type="double" value="3.0"/>
      <param name="imu_angular_velocity_error" type="double" value="0.0015"/>
      <param name="encoder_error" type="double" value="0.03"/>
      <param name="GPS_yaw_estimation_error" type="double" value="0.0050"/>
      <!--0.l0035-->
    </node>
</launch>
